src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -106,6 +106,113 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             expect(wrapper.find('ContentExplorerSelectAll').prop('isSelectAllChecked')).toEqual(isSelectAllChecked);
         });
 
+        test('should render ContentExplorerIncludeSubfolders with canIncludeSubfolders = true and isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const canIncludeSubfolders = true;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.exists('ContentExplorerIncludeSubfolders')).toBeTruthy();
```

Comment:
when using `exists`, we can be more explicit about the value: `expect(wrapper.exists('ContentExplorerIncludeSubfolders')).toBe(true);`